react-native-libraries.json

```
@@ -3959,5 +3959,337 @@
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/vinzscam/react-native-file-viewer",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-file-viewer",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/heyman333/react-native-responsive-fontsize",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-responsive-fontsize",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/aMarCruz/react-native-text-size",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-text-size",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/agentcooper/react-native-hypertext",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-hypertext",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/turley/react-native-asciimage",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-asciimage",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/taskrabbit/react-native-parsed-text",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-parsed-text",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/corymsmith/react-native-fabric",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-fabric",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/uxcam/react-native-ux-cam",
+    "ios": true,
+    "android": true,
+    "npmPkg": "",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/quipper/react-native-td",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-td",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/flurry/react-native-flurry-sdk",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-flurry-sdk",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/Starotitorov/react-native-deep-link",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-deep-link",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/necolas/react-native-web",
+    "web": true,
+    "npmPkg": "react-native-web",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/ptmt/react-native-macos",
+    "macos": true,
+    "npmPkg": "react-native-macos",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/Microsoft/reactxp",
+    "ios": true,
+    "android": true,
+    "web": true,
+    "npmPkg": "reactxp",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/microsoft/react-native-windows",
+    "windows": true,
+    "npmPkg": "react-native-windows",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/surialabs/react-native-geo-fencing",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-geo-fencing",
+    "unmaintained": false
```

```suggestion
    "unmaintained": true
```

not updated since 2016